web/css/index.css

```
@@ -17,14 +17,20 @@ body {
 
 }
 
+#left-panel {
+    height: 100vh;
+    overflow: hidden;
+}
+
 #agents {
 
 	transition: transform 0.25s;
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)